packages/plugin-awesome/src/charts/getPieChart.ts

```diff
@@ -0,0 +1,43 @@
+import type { Statistic, TestStatus } from "@allurereport/core-api";
```

Comment:
Don't we need to move these files to `web-components` too and re-use them across awesome and dashboard plugins?